src/pages/book/publish.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+// Components
+import BookCard from "../../components/book/bookCard";
+import PostBook from "../../components/book/PostBook";
+// Util
+import ScreamSkeleton from "../../util/ScreamSkeleton";
+// MUI Core
+import Grid from "@material-ui/core/Grid";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import DialogActions from "@material-ui/core/DialogActions";
+
+// Redux
+import { connect } from "react-redux";
+// import { getScreams } from "../../redux/actions/dataActions";
+import { getBookData } from "../../redux/actions/dataActions";
+import axios from "axios";
+
+const styles = (theme) => ({
+  ...theme.spread,
+  inProgress: {
+    marginBottom: "5em",
+  },
+  inProgressTitle: {
+    color: "#1c2a48",
+    fontWeight: "bolder",
+    fontFamily: "Times New Roman",
+    textShadow: "2px 2px #fff",
+  },
+  bookCardContainer: {
+    display: "flex",
+  },
+  publishHeader: {
+    background: "red",
+    width: "100%",
+  },
+  publishBody: {
+    display: "flex",
+  },
+});
+class bookreview extends Component {
+  state = {
+    screamIdParam: null,
+    open: false,
+  };
+  handleOpen = () => {
+    this.setState({ open: true });
+  };
+  handleClose = () => {
+    this.setState({ open: false });
+  };
+  componentDidMount() {
+    const handle = this.props.match.params.handle;
+    const screamId = this.props.match.params.screamId;
+
+    if (screamId) this.setState({ screamIdParam: screamId });
+
+    this.props.getBookData(handle);
+    axios
+      .get(`/user/${handle}`)
+      .then((res) => {
+        this.setState({
+          profile: res.data.user,
+        });
+      })
+      .catch((err) => console.log(err));
+  }
+
+  render() {
+    const { screams, loading } = this.props.data;
+    const { classes } = this.props;
+    const { screamIdParam } = this.state;
+    let recentScreamsMarkup = loading ? (
+      <ScreamSkeleton />
+    ) : screams === null ? (
+      <p>No screams from this user</p>
+    ) : !screamIdParam ? (
+      screams.map((scream) => (
+        <BookCard
```

Comment:
I think you need to fix some CSS here. The ENTIRE book card does not respond. You can only trigger the  on click action when you press the top half of the card